src/services/ResourcesService.js

```diff
@@ -0,0 +1,31 @@
+import axios from "@/axios.config";
+import { showNotification } from "@/utils/notification.js";
+
+async function fetchData(endpoint) {
+    try {
+        const response = await axios.get(endpoint);
+        if (response.status === 200) {
+            showNotification("Successfully retrieved!", "success");
```

Comment:
Let's remove the notification on success. Success is seen by the data loading to the table.